### PR TITLE
Css enhancement

### PR DESCRIPTION
**Pull request by MoiTux**
some modification to the css, to be more readable
see the current version:
 - http://status.gandi.net/timeline/events/255
 - http://status.gandi.net/api/events/255

and compare with the screenshot
front
api

### CODE REVIEW

**Comment by aegiap** [comment]
@MoiTux this seems fine, thank you.

File: baobab.front/less/event.less [hunk 1]
```diff
@@ -216,6 +219,10 @@ body > article > div {
     }
   }
 
+  p {
+    padding-top: 5px;
```
Comment (yanndinendal) [inline baobab.front/less/event.less]: plutôt margin (à l'extérieur de l'élément plutot qu'à l'intérieur)

**Comment by yanndinendal** [comment]
:+1:

**Comment by KillianKemps** [comment]
:+1: 
Concerning the timeline:
Not bad, why not add a line-height property to add some space in between the lines in each paragraph.

Add a bit more margin after the list to have more space above the last line.

**Comment by MoiTux** [comment]
@KillianKemps here are your modification:
front-v2

**Comment by KillianKemps** [comment]
@MoiTux nice! :+1:

**Pull request merged**